.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
the first thing you ever need to do is make the `.gitignore` file. Otherwise, all the node_modules will be uploaded to github. Look at your repo you will see you uploaded your node_modules on accident